### ISSUE TITLE
Formatting looks weird in the F1 Bonsai menu

### ISSUE THREAD

**Issue by cjsha** [open-ephys/bonsai-onix1-docs#16]
The tables don't really respond how they should when the height:width aspect ratio of the docs page is so huge. It looks fine in a typical desktop browser window, but ugly in the Bonsai browser assuming people use as demonstrated below

![image](https://github.com/user-attachments/assets/e570a580-6fe8-4e0c-847a-f29f9d89f65c)

Also, side-by-side with the text in Bonsai, the text on the website feels too big.

**Comment by jonnew** [comment]
I agree. However, because this is just a browser, the user can zoom out with Ctrl + mousewheel.  It would be better if the default was a bit smaller but the zoom functionality is actually pretty smooth.

**Comment by jonnew** [comment]
One solution for this is to provide docs on using the docs. For instance in the getting started pages, a page on using F1, expanding or undocking the F1-browser from bonsai, and copy pasting workflows. This is a situation where GIFs would be useful. 

- Click node, hit F1 GIF
- Undock and expand help browser GIF
- Copy paste example workflow GIF
